src/constants.py

```diff
@@ -45,14 +63,24 @@
     'mobile': 'Mobile Payments',
     'swipes': 'Meal Plan - Swipe'
 }
+SCHOOL_BREAKS = {
+  'fall': '10/6/18-10/9/18',
+  'thanksgiving': '11/21/18-11/25/18',
+  'finals_winter': '12/5/18-12/15/18',
+  'winter': '12/16/18-1/21/19',
+  'february': '2/23/19-2/26/19',
+  'spring': '3/30/19-4/7/19',
+  'finals_spring': '5/8/19-5/18/19',
+  'summer': '5/19/19-8/28/19'
```

Comment:
not sure what this is used for, but shouldn't this be school year independent? else this should be added to documentation as well, that it needs to be updated yearly

Reply:
this is used for determining whether or not a swipe event occurs during regular school conditions, or over a break.  This discrepancy is used to keep the wait time estimates as accurate as possible.  You are correct in that this would require updates yearly and I want to find a way to have it auto update by webscraping or something @young-k

Reply:
I could do the web scraping or point you in the right direction! I wrote a scraper for Uplift so we can talk about this